tests/plugins/subcommands/doctor/test_cli.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+import pytest
+
+from conda.testing.helpers import run_inprocess_conda_command as run
+from conda.plugins.subcommands.doctor.health_checks import MISSING_FILES_SUCCESS_MESSAGE
+
+
+TEST_ENV_1 = "test-env-1"
+
+
+def test_conda_doctor_happy_path():
+    """Make sure that we are able to call the ``conda doctor`` command"""
+
+    out, err, exit_code = run(f"conda doctor")
```

Comment:
you don't need f string here for running `conda doctor` there are no params in the string.. testing the run `conda doctor` calling the command test may not be the only thing you should test this test an function within for the test to be accurate..